Sources/Styleguide/PointFreeBaseStyles.swift

```diff
@@ -121,29 +123,54 @@ extension Class {
 extension Class.pf {
   public enum components {
 
-    public enum buttons {
-      public static let base =
-        baseButtonClass
-          | Class.type.medium
-          | Class.h5
-          | Class.padding([.mobile: [.leftRight: 2]])
-          | Class.border.rounded.all
+    public enum Color {
+      case black
+      case purple
+      case white
+    }
 
-      public static let purple =
-        base
-          | Class.pf.colors.link.white
-          | Class.pf.colors.bg.purple
+    public enum Size {
+      case small
+      case regular
+      case large
+    }
 
-      public static let black =
-        base
-          | Class.pf.colors.link.white
-          | Class.pf.colors.bg.black
+    public static func button(color: Color, size: Size = .regular) -> CssSelector {
```

Comment:
slightly refactored buttons so that it's easier to have small/med/large of different colors